src/airflow/dags/airnow.py

```diff
@@ -0,0 +1,153 @@
+from airflow.decorators import dag, task
+
+from airqo_etl_utils.airflow_custom_utils import AirflowUtils
+
+from airqo_etl_utils.airnow_api import AirNowApi
+
+from airqo_etl_utils.constants import DataSource
+
+
+@dag(
+    "Airnow-Historical-Bam-Data",
+    schedule=None,
+    default_args=AirflowUtils.dag_default_configs(),
+    catchup=False,
+    tags=["bam", "airnow", "historical"],
+)
+def airnow_bam_historical_data():
+    import pandas as pd
+
+    @task()
+    def extract_bam_data(**kwargs):
+        from airqo_etl_utils.date import DateUtils
+
+        start_date_time, end_date_time = DateUtils.get_dag_date_time_values(**kwargs)
+        from airqo_etl_utils.airnow_utils import AirnowDataUtils
+
+        airnow_api = AirNowApi()
+        networks = airnow_api.get_networks()
+        extracted_bam_data = pd.DataFrame()
+
+        for network in networks:
+            if network["data_source"] == DataSource.AIRNOW:
+                network_data = AirnowDataUtils.extract_bam_data(
+                    api_key=network["api_key"],
+                    start_date_time=start_date_time,
+                    end_date_time=end_date_time,
+                )
+                extracted_bam_data = pd.concat(
+                    [extracted_bam_data, network_data], ignore_index=True
+                )
+
+        return extracted_bam_data
+
+    @task()
+    def process_data(airnow_data: pd.DataFrame):
+        from airqo_etl_utils.airnow_utils import AirnowDataUtils
+
+        return AirnowDataUtils.process_bam_data(data=airnow_data)
```

Comment:
@BenjaminSsempala process_bam_data requires two positional arguments but you provided one. How will this task run successfully??